forte/data/data_store.py

```diff
@@ -187,6 +197,76 @@ def _new_tid(self) -> int:
         r"""This function generates a new ``tid`` for an entry."""
         return uuid.uuid4().int
 
+    def _get_type_info(self, type_name: str) -> Dict[str, int]:
+        """
+        Add a new type input_entry_class into ``_type_attributes``.
+        This method will get all attributes of a type and assign index to them.
+        Args:
+            type_name (str): A string or class type representing a class.
```

Comment:
"or class type"? did we take that? if so, the type annotation is wrong. 

and then, remember we should use "full qualified name"